events/wc_undead_events.txt

```diff
@@ -2134,6 +2134,68 @@ character_event = {
 	}
 }
 
+#####################
+# End of the Scourge
+#####################
+narrative_event = {
+	id = WCUND.1010
+	title = EVTTITLE_WCUND_1010
+	desc = EVTDESC_WCUND_1010
+	picture = GFX_evt_scourge_army
+	border = GFX_event_narrative_frame_war
+
+	is_triggered_only = yes
+	
+	major = yes
+	major_trigger = {
+		ai = no
+	}
+	show_root = yes
+	hide_from = yes
+	hide_new = yes
+	
+	trigger = {
+		has_landed_title = e_scourge
+	}
+
+	immediate = {
+		activate_title = { title = e_scourge status = no }
+		e_scourge = { destroy_landed_title = THIS }
+	}
+
+	# It's not over yet!
+	option = {
+		trigger = {
+			OR = {
+				true_religion = death_god
+				true_religion = burning_legion_religion
+			}
+		}
+		name = ITS_NOT_OVER_YET
+	}
+	# We can sleep peacefully now
+	option = {
+		trigger = {
+			NOT = {
+				true_religion = death_god
+				true_religion = burning_legion_religion
+			}
+			is_aggressive_trigger = no
+		}
+		name = WE_CAN_SLEEP_PEACEFULLY_NOW
+	}
+	# Who cares?
+	option = {
+		trigger = {
+			NOT = {
+				true_religion = death_god
+				true_religion = burning_legion_religion
+				is_aggressive_trigger = no
+			}
+		}
+		name = WHO_CARES
+	}
```

Comment:
Are you sure that you want to have the same option three times with only localization difference?

https://ck2.paradoxwikis.com/Event_modding#Option_Name

Reply:
Yes, it's easier to debug. You see +1 option when you did something wrong.